src/FingerprintPro.ServerSdk/Model/IpInfoResultV4.cs

```diff
@@ -111,6 +120,11 @@ public bool Equals(IpInfoResultV4 input)
                 this.Asn.Equals(input.Asn))
                 ) &&
                 (
+                this.Datacenter == input.Datacenter ||
+                (this.Datacenter != null &&
+                this.Datacenter.Equals(input.Datacenter))
+                ) &&
+                (
                 this.DataCenter == input.DataCenter ||
```

Comment:
@ilfa Are we sure we want to break the currently working `dataCenter`? 

Currently: 
* `Console.WriteLine(event.Products.IpInfo.Data.V4.DataCenter)` works
* `Console.WriteLine(event.Products.IpInfo.Data.V4.Datacenter)` does not exist 

After this change: 
* `Console.WriteLine(events.Products.IpInfo.Data.V4.DataCenter)` doesn't work (is null)
* `Console.WriteLine(events.Products.IpInfo.Data.V4.Datacenter)` works

Why not map `dataCenter` the same as `datacenter` to avoid the breaking change?